pandas/_libs/tslib.pyx

```diff
@@ -83,6 +84,11 @@ def ints_to_pydatetime(const int64_t[:] arr, object tz=None, object freq=None,
          convert to this timezone
     freq : str/Offset, default None
          freq to convert
+    fold : bint, default is 0
```

Comment:
even though these are not public doc-strings, you can add a versionadded 1.1 tag

Reply:
Done.